### PR TITLE
ADEN-8806 Use Instant Config. Use wgAdDriverUnstickHiViLeaderboardAft…

### PR DESCRIPTION
**Pull request by jacekbj**
…erTimeoutCountries to control HiVi LB unstick time.

### CODE REVIEW

File: extensions/wikia/AdEngine3/src/setup.js [hunk 1]
```diff
@@ -175,15 +213,17 @@ function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = tru
...
-		context.set('custom.rubiconInFV',
-			isGeoEnabled('wgAdDriverRubiconPrebidCountries') && hasFeaturedVideo);
+		context.set(
+			'custom.rubiconInFV',
+			isGeoEnabled('wgAdDriverRubiconPrebidCountries') && hasFeaturedVideo,
+		);
 		context.set('custom.isCMPEnabled', true);
 
-		if (!isGeoEnabled('wgAdDriverLkqdOutstreamCountries')) {
+		if (!(isGeoEnabled('wgAdDriverLkqdOutstreamCountries'))) {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/src/setup.js]: Why additional parentheses? :)

File: extensions/wikia/AdEngine3/src/setup.js [hunk 2]
```diff
@@ -29,18 +54,23 @@ function updateWadContext() {
...
-	return utils.geoService.isProperGeo(instantGlobals.get(key), key);
-}
-
-function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = true) {
+async function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = true) {
 	const showAds = window.ads.context.opts.showAds;
 
 	context.extend(basicContext);
+
+	// set
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/src/setup.js]: Isn't set(... descriptive enough? :p

Comment (jacekbj) [reply extensions/wikia/AdEngine3/src/setup.js]: Old and forgotten comment 🕵

**Review by dmnsobczak — APPROVED** [review]

**Review by dmnsobczak — APPROVED** [review]

**Review by bckatarzyna — APPROVED** [review]

**Comment by jacekbj** [comment]
Please 👍 👎 commit "ADEN-8806 Add backward compatibility."

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**